recipes/perl-uri/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "perl-uri" %}
+{% set version = "5.17" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://cpan.metacpan.org/authors/id/O/OA/OALDERS/URI-5.17.tar.gz
+  sha256: 5f7e42b769cb27499113cfae4b786c37d49e7c7d32dbb469602cd808308568f8
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}        # [not win]
+    - {{ compiler('m2w64_c') }}  # [win]
+    - make
+  host:
```

Comment:
Okay, you have a lot of dependencies listed here. However, almost all of them are actually core modules and vendored with Perl already. These don't need to be added as requirements. 

A simple way to check for non-core dependencies is a short script like this (you may need to install `cpanm` from [App::Cpanminus](https://metacpan.org/pod/App::cpanminus) and `corelist` from [Module::CoreList](https://metacpan.org/pod/distribution/Module-CoreList/lib/Module/CoreList.pod))

```bash
cpanm --showdeps URI | 
    tail -n+4 | 
    sed 's/~.*$//' |  
    while read m; do 
        corelist "$m" |  grep 'was not in CORE'
    done 
```

Output:

```
perl was not in CORE (or so I think)
Test::Needs was not in CORE (or so I think)
Test::Warnings was not in CORE (or so I think)
Test::Fatal was not in CORE (or so I think)
```

Ignoring the first line, we find only 3 dependencies: `perl-test-needs`, `perl-test-warnings`, and `perl-test-fatal`. You can remove the other ones.